notebooks/curation_api/create_collection_dev_R.ipynb

```diff
@@ -0,0 +1,213 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c5490d93",
+   "metadata": {},
+   "source": [
+    "# Create a new private Collection"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "74560383",
+   "metadata": {},
+   "source": [
+    "The script in this notebook creates a new private Collection.\n",
+    "\n",
+    "In order to use this script, you must have a Curation API key (obtained from upper-righthand dropdown in the Data Portal UI after logging in)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "928aa50e",
+   "metadata": {},
+   "source": [
+    "#### <font color='#bc00b0'>Please fill in the required values:</font>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2c9a0189",
+   "metadata": {},
+   "source": [
+    "<font color='#bc00b0'>(Required) Provide the path to your api key file</font>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a4b6087b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "api_key_file <- \"path/to/api-key.txt\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "edfb1272",
+   "metadata": {},
+   "source": [
+    "### Import dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1cff472c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "library(\"readr\")\n",
+    "library(\"httr\")\n",
+    "library(\"stringr\")\n",
+    "library(\"rjson\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "285a1ca4",
+   "metadata": {},
+   "source": [
+    "### Specify domain (and API url)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bea213f4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "domain_name <- \"cellxgene.dev.single-cell.czi.technology\"\n",
```

Comment:
This should point to prod by default. We don't want to advertise the dev environment to the curators.

Reply:
This also applies to the python note book.

Reply:
or we need to tell the users where to find the domain name they need. It's not obvious that they can use the same domain from the frontend.

Reply:
agreed, will change

Reply:
done